Content.Server/GameObjects/Components/Construction/ConstructionComponent.cs

```diff
@@ -144,46 +144,10 @@ bool TryProcessStep(ConstructionStep step, IEntity slapped)
                 case ConstructionStepTool toolStep:
                     if (!slapped.TryGetComponent<ToolComponent>(out var tool))
                         return false;
-                    switch (toolStep.Tool)
-                    {
-                        case Tool.Crowbar:
-                            if (tool.Behavior == Tool.Crowbar)
-                            {
-                                tool.PlayUseSound();
-                                return true;
-                            }
-                            return false;
-                        case Tool.Welder:
-                            if (tool.Behavior == Tool.Welder && tool.TryWeld(toolStep.Amount))
-                            {
-                                tool.PlayUseSound();
-                                return true;
-                            }
-                            return false;
-                        case Tool.Wrench:
-                            if (tool.Behavior == Tool.Wrench)
-                            {
-                                tool.PlayUseSound();
-                                return true;
-                            }
-                            return false;
-                        case Tool.Screwdriver:
-                            if (tool.Behavior == Tool.Screwdriver)
-                            {
-                                tool.PlayUseSound();
-                                return true;
-                            }
-                            return false;
-                        case Tool.Wirecutter:
-                            if (tool.Behavior == Tool.Wirecutter)
-                            {
-                                tool.PlayUseSound();
-                                return true;
-                            }
-                            return false;
-                        default:
-                            throw new NotImplementedException();
-                    }
+                    if (toolStep.Tool != tool.Behavior) return false;
+                    tool.PlayUseSound();
+                    return true;
```

Comment:
You removed `tool.TryWeld(toolStep.Amount)`, is this on purpose?

Reply:
Not at all, my bad. Fixed now.